src/apis/routes/orders.routes.js

```diff
@@ -0,0 +1,86 @@
+import { body, param } from "express-validator";
+import ordersController from "../controllers/orders.controller";
+import adminAuthorization from "../middlewares/auth";
+/**
+ *
+ * @param {import('express').Router} router
+ */
+
+const ordersRoutes = (router) => {
+  // create new order with customerId,status,shippingFee,total,couponId
+  router.post(
+    "/create-order",
```

Comment:
use path `/orders` method `post` for create new order

Reply:
Done